cli/fixtures/gpte/OCP_CLIENTVM/description.adoc

```diff
@@ -0,0 +1 @@
+= foobar
```

Comment:
It seems that this has no function in example? I think either not include at all or at least make it look a bit more like valid description?